hetio/abbreviation.py

```diff
@@ -111,7 +111,7 @@ def metaedges_from_metapath(abbreviation, standardize_by=None):
     if isinstance(standardize_by, hetio.hetnet.MetaGraph):
         metapath = standardize_by.metapath_from_abbrev(abbreviation)
         return [metaedge.get_standard_abbrev() for metaedge in metapath]
-    pattern = regex.compile('(?<=^|[a-z<>])[A-Z]+[a-z<>]+[A-Z]+')
+    pattern = regex.compile('(?<=^|[a-z<>])[A-Z0-9]+[a-z<>]+[A-Z0-9]+')
```

Comment:
Currently this treats all digits as metanode abbreviations... i.e. 0-9 is always uppercase. My idea was require the first character of an abbreviation to be a letter, then allow numbers to follow. The first letter would set the case and thus whether it's a metanode or metaedge.

With the current implementation it is not possible to add numbers to metaedge abbreviations, which could be helpful for some users.

Reply:
is it sufficient to check and disallow if first element of metanode is a digit? 07027e2

Reply:
I would prefer for the regex to be aware that metanodes cannot start with a digit. How about changing the node regex to `[A-Z][A-Z0-9]*`? I think the consistency will be better in the longrun.